src/components/WalletSettings/SetTorusKey/SetTorusKey.vue

```diff
@@ -0,0 +1,114 @@
+<template>
+  <div :class="$vuetify.breakpoint.xsOnly ? 'pt-5' : 'py-5 px-4'">
+    <v-list dense outlined class="pa-0 account-list mb-2">
```

Comment:
put v-if="torusWallets.length > 0" cause this list will be empty in binance.tor.us